index.js

```diff
@@ -2,8 +2,11 @@
 var objectAssign = require('object-assign');
```

Comment:
I did not make any change to the core implementation, IMO this should be resolved in a separate issue/pr, what do you think?